### ISSUE TITLE
RotaryCraft and ReactorCraft textures

### ISSUE THREAD

**Issue by AlexPlus117** [F32Organization/Faithful32-1.7.10#359]
Hi, could you add missing textures of RotaryCraft and ReactorCraft mod, please? Thank you in advance.

**Comment by Evolution0** [comment]
Yeah pretty much all of Reika's mods are lacking support atm, I started on it but BC7 kinda grabbed my attention.

I'll probably get back to it soon though seeing as I now have them in my custom pack.
